test/clj_sqs_extended/core_test.clj

```diff
@@ -8,52 +8,67 @@
+    :by    "Vidiu Platon"}
+   {:quote "If you can't beat your computer at chess, try kickboxing."
+    :by    "Claudio E. Montero"}
+   {:quote "I would rather be without a state than without a voice."
+    :by    "Edward Snowden"}])
 
 (def ^:private test-message-larger-than-256kb
   (helpers/random-string-with-length 300000))
 
 (deftest can-receive-message
   (testing "Sending/Receiving basic maps"
     (let [client (fixtures/localstack-sqs)
-          url (fixtures/test-queue-url)]
+          url (fixtures/test-queue-url)
+          test-message (first test-messages)]
       (doseq [format [:transit :json]]
-        (sqs-ext/send-message client url test-message-1 {:format format})
+        (sqs-ext/send-message client url test-message {:format format})
         (let [message (sqs-ext/receive-message client url {:format format})]
-          (is (= test-message-1 (:body message))))))))
+          (is (= test-message (:body message))))))))
 
 (deftest can-auto-delete-message
   (testing "Auto-Deleting a single message after receiving it"
     (let [client (fixtures/localstack-sqs)
-          url (fixtures/test-queue-url)]
+          url (fixtures/test-queue-url)
+          test-message (rand-nth test-messages)]
       (sqs-ext/purge-queue client url)
-      (sqs-ext/send-message client url test-message-1)
+      (sqs-ext/send-message client url test-message)
       (let [message (sqs-ext/receive-message
                       client
                       url
                       {:auto-delete true})]
-        (is (= test-message-1 (:body message)))
+        (is (= test-message (:body message)))
         (is (= 0 (helpers/get-total-message-amount-in-queue client url)))))))
 
 (deftest can-receive-fifo-messages
   (testing "Receiving multiple messages from FIFO queue in correct order"
     (let [client (fixtures/localstack-sqs)
           url (fixtures/test-fifo-queue-url)]
       (doseq [format [:transit :json]]
-        (let [group-id-1 (str format "-1")
-              group-id-2 (str format "-2")]
-          (sqs-ext/send-fifo-message client url test-message-1 group-id-1 {:format format})
-          (sqs-ext/send-fifo-message client url test-message-2 group-id-2 {:format format}))
+        (sqs-ext/purge-queue client url)
+        (doseq [message test-messages]
+          (sqs-ext/send-fifo-message client
+                                     url
+                                     message
+                                     (helpers/random-string-with-length 5)
+                                     {:format format}))
         (let [message (sqs-ext/receive-message client url {:format format})]
-          (is (= test-message-1 (:body message))))
+          (is (= (nth test-messages 0) (:body message))))
         (let [message (sqs-ext/receive-message client url {:format format})]
-          (is (= test-message-2 (:body message))))))))
+          (is (= (nth test-messages 1) (:body message))))
```

Comment:
this is repetitive. keep track of an `i` index in your doseq and so similar to above as L56 to iterate the reads through.